src/cluster_legacy.c

```diff
@@ -5001,7 +5002,7 @@ void clusterCron(void) {
 
     /* Ping some random node 1 time every 10 iterations, so that we usually ping
      * one random node every second. */
-    if (!(iteration % 10)) {
+    if (server.debug_cluster_random_ping && !(iteration % 10)) {
```

Comment:
```suggestion
    if (!server.debug_disable_cluster_ping && !(iteration % 10)) {
```
I think this will come across as a debug feature that isn't normally enabled because of the name. I think we should invert the flag so it's explicitly disabling some functionality.